src/networks.json

```diff
@@ -1176,6 +1176,22 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2152": {
+    "key": "2152",
+    "name": "Findora Mainnet",
+    "shortName": "Findora",
+    "chainId": 2152,
+    "network": "mainnet",
+    "multicall": "0x319E6CCaA89C3dCe65946E759a32a09531E6b72f",
+    "rpc": [
+      "https://archive.prod.findora.org:8545/"
+    ],
+    "explorer": {
+      "url": "https://evm.findorascan.io"
+    },
+    "start": 3378569,
+    "logo": "ipfs://QmQpqPYEBgGGCoM46sCh2kgVQ5BYrM1JryLXkXMrNtE74S"
```

Comment:
Hi @ws4charlie Looks like the Image on IPFS is not pinned https://ipfs.io/ipfs/QmQpqPYEBgGGCoM46sCh2kgVQ5BYrM1JryLXkXMrNtE74S
Not resolving, try pinning it with a third party pinning provider, everything else looks good :)

Reply:
@ChaituVR Thank you for reporting. I just updated the link in my PR. Can you please try ipfs://QmXkneyRB6HbHTHRLCZpZqSsawiyJY7b2kZ2V8ydvKYAgv